clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5972,6 +5972,15 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
             << A->getAsString(Args) << A->getValue();
       else
         A->render(Args, CmdArgs);
+    } else if (Triple.isAArch64()) {
+      // "all" is not supported on AArch64 since branch relaxation creates new
```

Comment:
I think `Triple.isOSBinFormatELF()` is needed (just like for x86).

Reply:
Yep, added! I got so caught up writing a comment that I forgot to actually do what I created this PR to do...